git-host-info.js

```diff
@@ -76,7 +76,7 @@ gitHosts.gitlab = Object.assign({}, defaults, {
   domain: 'gitlab.com',
   treepath: 'tree',
   httpstemplate: ({ auth, domain, user, project, committish }) => `git+https://${maybeJoin(auth, '@')}${domain}/${user}/${project}.git${maybeJoin('#', committish)}`,
-  tarballtemplate: ({ domain, user, project, committish }) => `https://${domain}/${user}/${project}/repository/archive.tar.gz?ref=${maybeEncode(committish) || 'master'}`,
+  tarballtemplate: ({ domain, user, project, committish }) => `https://${domain}/${user}/${project}/repository/archive.tar.gz?sha=${maybeEncode(committish) || 'master'}`,
```

Comment:
will this both work when the params is a commit hash or a branch name as "master"?

Reply:
> will this both work when the params is a commit hash or a branch name as "master"?

ofc, see gitlab document: https://docs.gitlab.com/ee/api/repositories.html#get-file-archive

Reply:
great, thanks for the link